DESCRIPTION

```diff
@@ -38,11 +38,15 @@ Imports:
     renv,
     rlang,
     rmarkdown,
+    tidyr,
     tibble,
     withr
 Suggests: 
-    testthat
+    knitr,
```

Comment:
presumably this is only for the documentation and won't affect any code that would run on the website? At the moment the Docker image is locked in to a old version of R and specific versions of a few packages, including knitr <v1.34 because of requirements in create_interactive_report (see [create_interactive_report/issues/422](https://github.com/2DegreesInvesting/create_interactive_report/issues/422)

Reply:
I believe this is an artifact of using `usethis::use_article()`. Since the package is under "Suggests", it is not actually used or required by any part of the package, it is just an indication that `knitr` could be useful to the package:

<img width="646" alt="Screenshot 2021-11-14 at 22 05 42" src="https://user-images.githubusercontent.com/8741309/141698602-626e9c2d-8303-4501-a331-a7b189eacac3.png">
e

My guess is that this is included automatically, since articles are not actually bundled with the package (in comparison to vignettes). See here: https://usethis.r-lib.org/reference/use_vignette.html

Reply:
any idea what will happen when `devtools::load_all()` is called if `knitr` is not installed? like here https://github.com/2DegreesInvesting/PACTA_analysis/blob/675ed324da6a2e6315af2c36ab30336d1a891e49/web_tool_script_1.R#L3

Reply:
So it looks like `knitr` is a dependency of `rmarkdown`, and `rmarkdown` IS actually imported by this package. 
(i.e. `devtools::load_all()` will fail if `knitr` is not installed, but it's not because `knitr` is a suggested package).

Reply:
yeah, so `knitr` is installed in the current Docker image. It's stuck on version 1.33 or less though, so we have to be careful about what/how relies on it.

Reply:
yup, so again this is totally fine since it's `Suggests`. Nothing relies on it in any way.